docs/source/search-syntax.rst

```diff
@@ -77,6 +77,20 @@ For example:
   metrics."2019-04-02 error rate"
 
```

Comment:
Please provide an example with backticks.

Reply:
Added one underneath the Searching by MLflow Tags section.